pkg/version/version.go

```diff
@@ -1,26 +1,23 @@
 package version
 
 import (
-	"context"
-	"encoding/json"
+	"errors"
 	"fmt"
-	"io/ioutil"
-	"net/http"
 	"os"
-	"strings"
-	"time"
-
-	pb "github.com/linkerd/linkerd2/controller/gen/public"
 )
 
-// Version is updated automatically as part of the build process
+// This module depends only on channelVersion in the version package.
```

Comment:
yeah this is a poorly worded comment. you're correct everything is in the `version` package, i used the word `module` to try to convey "this file".

within the `version` package, we have these dependencies:

`channels.go` -> `channelversion.go`
`version.go` -> `channelversion.go`

i wrote these comments because the file names feel a bit ambiguous. would it be clearer to simply state those dependencies in each of the 3 files?

Reply:
Sure -- yeah, I think my confusion was that I typically think of "module" as synonymous with "package", but you're using it to mean "file".

If you're really concerned about strict dependencies, you could consider moving `channelversion.go` to a separate package. Then it would be disallowed from importing the version package due to circular dependencies.

Reply:
less concerned around strict dependencies, but instead was focused on orienting the reader, which seemed to have backfired. =)